core/controllers/editor.py

```diff
@@ -706,6 +706,26 @@ def get(self, exploration_id, exploration_version):
             current_exploration.states))
 
 
+class ExplorationStatisticsHandlerV2(EditorHandler):
```

Comment:
Same as before, call this ExplorationStatisticsHandler and the other one OldExplorationStatisticsHandler. Makes deprecation easier in the future.

Reply:
Done.